source/Classroom/Basics/Billing/how_to_read_your_invoice.md

```diff
@@ -1,86 +1,69 @@
 ---
 seo:
-  title: How to read your invoice
-  description: How to read your invoice
+  title: Reading your invoice
+  description: Reading your invoice
   keywords: invoice, bill, receipt, refund, line, item, marketing, campaigns, charge, contacts, usage, overage
-title: How to read your invoice
+title: Reading your invoice
 weight: 0
 layout: page
 zendesk_id: 
 navigation:
   show: true
 ---
-When you receive your invoice on the 1st of each month, you might have questions. This article aims to clarify what your invoices mean, and to help you learn more about what you are paying for.
 
```

Comment:
I don't know that I would cast the invoice in a negative light right off the bat. Maybe something like:

"Your SendGrid invoice contains a lot of information, which may be difficult to read. Here's information on how to read your invoice:"